src/OpenSage.Game/Gui/Apt/ActionScript/Library/Builtin.cs

```diff
@@ -162,7 +178,7 @@ private static void GotoAndStop(ObjectContext ctx, Value[] args)
                     throw new InvalidOperationException("Can only jump to labels or frame numbers");
                 }
 
-                si.Stop();
+                si.Stop(true);
```

Comment:
`gotoAndStop` wasn't playing the frame where it went to. Added a PENDING_STOP type